ports/servo/browser.rs

```diff
@@ -315,6 +317,18 @@ impl Browser {
 
 }
 
+#[cfg(any(target_os = "macos", target_os = "linux", target_os = "windows"))]
+fn display_alert_dialog(message: &str) {
+    if !opts::get().headless {
+        tinyfiledialogs::message_box_ok("Alert!", message, MessageBoxIcon::Warning);
```

Comment:
You probably want to run that in a different thread. See `platform_get_selected_devices` for example.